install_pim/system_requirements/system_requirements.rst.inc

```diff
@@ -30,17 +30,19 @@ The web server will also need the following libraries and modules:
 
 **PHP required modules and configuration**
 
-+-------------+---------------------------+
-| PHP         | 5.6                       |
-+-------------+---------------------------+
-| php5-curl   | No specific configuration |
-+-------------+---------------------------+
-| php5-gd     | No specific configuration |
-+-------------+---------------------------+
-| php5-intl   | No specific configuration |
-+-------------+---------------------------+
-| php5-apcu   | apc.enable_cli=1          |
-+-------------+---------------------------+
++---------------+---------------------------+
+| PHP           | 7.1                       |
++---------------+---------------------------+
+| php7.1-curl   | No specific configuration |
++---------------+---------------------------+
+| php7.1-gd     | No specific configuration |
++---------------+---------------------------+
+| php7.1-intl   | No specific configuration |
++---------------+---------------------------+
+| php7.1-bcmath | No specific configuration |
++---------------+---------------------------+
+| php-apcu      | apc.enable_cli=1          |
```

Comment:
should I list all the extensions we require?
because I think there are more than that